### PR TITLE
feat(vpn): vpn_gateway support update enterprise_project_id

### PR DESCRIPTION
**Pull request by wangyuancheng01**
**What this PR does / why we need it**: vpn_gateway support update enterprise_project_id

**Which issue this PR fixes**:
*(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*
fixes #xxx

**Special notes for your reviewer**:

**Release note**:

```release-note

```

## PR Checklist

* [x] Tests added/passed.
* [x] Documentation updated.
* [x] Schema updated.

## Acceptance Steps Performed

```
make testacc TEST="./huaweicloud/services/acceptance/vpn" TESTARGS="-run TestAccGateway_UpdateWithEpsId"
...
==> Checking that code complies with gofmt requirements...
TF_ACC=1 go test ./huaweicloud/services/acceptance/vpn -v -run TestAccGateway_UpdateWithEpsId -timeout 360m -parallel 4
=== RUN   TestAccGateway_UpdateWithEpsId
=== PAUSE TestAccGateway_UpdateWithEpsId
=== CONT  TestAccGateway_UpdateWithEpsId
--- PASS: TestAccGateway_UpdateWithEpsId (301.81s)
PASS
ok      github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/services/acceptance/vpn       301.852s
```

### CODE REVIEW

File: huaweicloud/services/acceptance/vpn/resource_huaweicloud_vpn_gateway_test.go [hunk 1]
```diff
@@ -314,11 +314,12 @@ func testGateway_update(name string) string {
...
 
 resource "huaweicloud_vpn_gateway" "test" {
-  name               = "%s-update"
-  vpc_id             = huaweicloud_vpc.test.id
-  local_subnets      = [huaweicloud_vpc_subnet.test.cidr, "192.168.2.0/24"]
-  connect_subnet     = huaweicloud_vpc_subnet.test.id
-  availability_zones = [
+  name                  = "%s-update"
+  vpc_id                = huaweicloud_vpc.test.id
+  enterprise_project_id = "88f889c7-270e-4e77-8230-bf7db08d9b0e"
```
Comment (Jason-Zhang9309) [inline huaweicloud/services/acceptance/vpn/resource_huaweicloud_vpn_gateway_test.go]: use environment variable

Comment (wangyuancheng01) [reply huaweicloud/services/acceptance/vpn/resource_huaweicloud_vpn_gateway_test.go]: down

File: huaweicloud/services/acceptance/vpn/resource_huaweicloud_vpn_gateway_test.go [hunk 2]
```diff
@@ -309,16 +365,42 @@ resource "huaweicloud_vpn_gateway" "test" {
...
-  name               = "%s-update"
-  vpc_id             = huaweicloud_vpc.test.id
-  local_subnets      = [huaweicloud_vpc_subnet.test.cidr, "192.168.2.0/24"]
-  connect_subnet     = huaweicloud_vpc_subnet.test.id
-  availability_zones = [
+  name                  = "%s-update"
+  vpc_id                = huaweicloud_vpc.test.id
+  local_subnets         = [huaweicloud_vpc_subnet.test.cidr, "192.168.2.0/24"]
+  connect_subnet        = huaweicloud_vpc_subnet.test.id
+  availability_zones    = [
```
Comment (Jason-Zhang9309) [inline huaweicloud/services/acceptance/vpn/resource_huaweicloud_vpn_gateway_test.go]: remove extral blanks

Comment (wangyuancheng01) [reply huaweicloud/services/acceptance/vpn/resource_huaweicloud_vpn_gateway_test.go]: down

**Pull request merged**